.github/workflows/pr_validation.yml

```diff
@@ -50,5 +50,5 @@ jobs:
         with:
           cov-omit-list: tests/*, *__init__*, *__version__*, *__main__*, *exceptions*
           cov-threshold-single: 70
-          cov-threshold-total: 85
+          cov-threshold-total: 80
```

Comment:
moving into the bad direction  --> understandable for 0.0.2 tightness, but should have an issue to pump it up in 0.0.3 ?

Reply:
I'll have to look into all the missing lines of all files to make the tests cover more.

Reply:
a new issue has been made and put into milestone 0.0.3 to cover this https://github.com/vliz-be-opsci/py-trav-harv/issues/47